Library/Homebrew/dev-cmd/release.rb

```diff
@@ -70,12 +69,15 @@ def release
     if args.major? || args.minor?
       latest_major_minor_version = "#{latest_version.major}.#{latest_version.minor.to_i}.0"
       ohai "Release notes since #{latest_major_minor_version} for #{new_version} blog post:"
-      # release notes without username suffix or dependabot bumps
-      puts ReleaseNotes.generate_release_notes(latest_major_minor_version, "origin/HEAD")
-                       .lines
-                       .reject { |l| l.include?(" (@Homebrew)") }
-                       .map { |l| l.gsub(/ \(@[\w-]+\)$/, "") }
-                       .sort
+      # release notes without usernames, new contributors, or extra lines
+      blog_post_notes = GitHub.generate_release_notes("Homebrew", "brew", new_version,
+                                                      previous_tag: latest_major_minor_version)["body"]
```

Comment:
Do we want to handle if `["body"]` doesn't exist? Alternatively, shall we just return that from the `generate_release_notes` method so that `nil` or `""` can be better handled here?